### PR TITLE
Add release notes for the new metrics endpoint

### LINKED ISSUE #824: Deprecation notice for MCAC in v1.5.0
Add a deprecation notice in the release notes for MCAC in v1.5.0, with the appropriate information

### PR DESCRIPTION
**Pull request by adejanovski**
**What this PR does**:
Adds a release notes entry for the new metrics endpoint.

**Which issue(s) this PR fixes**:
Fixes #824 

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: CHANGELOG/RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
 # k8ssandra-operator - Release Notes
 
+## v1.5.0
+
+
+### New Metrics Endpoint
+
+As of v1.5.0, we are introducing a new metrics endpoint which will replace MCAC in an upcoming release and is built directly in the Management API.
```
Comment (burmanm) [inline CHANGELOG/RELEASE-NOTES.md:8]: Do users know what MCAC is..?

Comment (adejanovski) [reply CHANGELOG/RELEASE-NOTES.md:8]: Well, we've always mentioned it as part of the main components in K8ssandra: https://docs.k8ssandra.io/
Maybe I should add a link to the github project?

**Review by emerkle826 — APPROVED** [review]
Approved with the one suggestion

**Comment by adejanovski** [comment]
> Approved with the one suggestion

Done ✅

**Pull request merged**